src/wp-admin/includes/image-edit.php

```diff
@@ -1026,12 +1061,177 @@ function wp_save_image( $post_id ) {
 		$meta['sizes'] = array_merge( $meta['sizes'], $img->multi_resize( $_sizes ) );
 	}
 
+	$resize_sizes = array();
+	foreach ( $old_metadata['sizes'] as $size_name => $size_details ) {
+		// If the target is 'nothumb', skip generating the 'thumbnail' size.
+		if ( 'nothumb' === $target && 'thumbnail' === $size_name ) {
+			continue;
+		}
+
+		if (
+			isset( $meta['sizes'][ $size_name ] ) && ! empty( $meta['sizes'][ $size_name ] ) &&
+			$meta['sizes'][ $size_name ]['file'] !== $old_metadata['sizes'][ $size_name ]['file']
+		) {
+			$resize_sizes[ $size_name ] = $meta['sizes'][ $size_name ];
+		}
+	}
+
+	$uploads_dir        = wp_get_upload_dir();
+	$allowed_mimes      = array_flip( wp_get_mime_types() );
+	$original_directory = pathinfo( $new_path, PATHINFO_DIRNAME );
+	$filename           = pathinfo( $new_path, PATHINFO_FILENAME );
+	$main_images        = array();
+	$subsized_images    = array();
+
+	$transforms = wp_upload_image_mime_transforms( $post_id, false );
```

Comment:
wp_upload_image_mime_transforms only takes a single parameter

Reply:
Oh! nice catch. It was in previous version of add/webp-uploads branch, now it is removed.